source/Halibut.OctopusSample/Halibut.OctopusSample.csproj

```diff
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <VersionPrefix>0.0.0</VersionPrefix>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFramework>net48</TargetFramework>
     <AssemblyName>Halibut.OctopusSample</AssemblyName>
     <OutputType>Exe</OutputType>
+    <UseWindowsForms>true</UseWindowsForms>
```

Comment:
What is this about?

Reply:
Had to add it to get the correct references for WinForms so the project would compile!!

Reply:
I'm not an expert here, google said this will fix it and it did 🤷

Reply:
Oh i see I don't know more so if it works :D. What does chatgpt say :p